### PR TITLE
Admin active alerts endpoints

### PR DESCRIPTION
**Pull request by evie-adenoyin**

### CODE REVIEW

File: src/pages/Admin/ActiveAlert/ActiveAlert.jsx [hunk 1]
```diff
@@ -74,34 +91,35 @@ const ActiveAlertDetails = (data) => {
                   </div>
                   <Select
                     className={classes.ActiveAlert__ActiveAlertModalFormSelect}
-                    defaultValue={data.data.shs[0].name}
+                    defaultValue={'data?.data?.active_alert'}
```
Comment (slimsolz) [inline src/pages/Admin/ActiveAlert/ActiveAlert.jsx:94]: Are you expecting the value from the backend?

Comment (evie-adenoyin) [reply src/pages/Admin/ActiveAlert/ActiveAlert.jsx:94]: yeah... once they update me with the extra data i'll take out the string values

Comment (slimsolz) [reply src/pages/Admin/ActiveAlert/ActiveAlert.jsx:94]: please remove it or comment it out then add some default text, it didn't look good during the demo

File: src/pages/Admin/ActiveAlert/ActiveAlert.jsx [hunk 2]
```diff
@@ -267,12 +322,26 @@ const ActiveAlert = () => {
...
+      await createAdminActiveAlerts(values)
+      setIsModalOpen(false)
+    } catch (err) {
+      let errorMsg = ''
+
+      if (err.status === 401) {
+        errorMsg += err?.data?.detail
+        setErrMsg(errorMsg)
+      } else if (err.status === 400) {
+        setErrMsg('Missing username or password')
```
Comment (slimsolz) [inline src/pages/Admin/ActiveAlert/ActiveAlert.jsx:336]:
```suggestion
        setErrMsg('invalid username or password')
```

Comment (evie-adenoyin) [reply src/pages/Admin/ActiveAlert/ActiveAlert.jsx:336]: I have changed it to the response from the backend... I suggest we just give them a correction... I could talk to Abram on that.

**Review by slimsolz — APPROVED** [review]
LGTM

**Pull request merged**